WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -139,6 +143,9 @@
     @Inject ViewModelProvider.Factory mViewModelFactory;
     private EditPostPublishSettingsViewModel mPublishedViewModel;
 
+    private final OnCheckedChangeListener onStickySwitchChangeListener =
```

Comment:
Checkstyle is suggesting that we adhere to the pattern of putting a `m` in front of the member variable. 
```suggestion
    private final OnCheckedChangeListener mOnStickySwitchChangeListener =
```

Reply:
Oh, now I see that the Checkstyle job failed (but I can't see the details in BuildKite). I guess I should've ran the checks locally. Thanks for pointing it out :)